File: examples-sdk/python/504-villog-benchmark-comparison.py
#!/usr/bin/env python3
"""504-villog-benchmark-comparison — Python SDK equivalent
Compile: vil compile --from python --input 504-villog-benchmark-comparison.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()

File: examples-sdk/typescript/504-villog-benchmark-comparison.ts
#!/usr/bin/env tsx
// 504-villog-benchmark-comparison — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 504-villog-benchmark-comparison.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();
